.github/workflows/license-checker.yml

```diff
@@ -4,12 +4,12 @@ on:
   push:
     branches: [main]
     paths:
-      - '**/pnpm-lock.yaml'
-      - '**/package.json'
+      - "**/pnpm-lock.yaml"
```

Comment:
arruma o format ai pra ficar no padrao aspas simples pls. Deve ter como forçar ser sempre um estilo pro projeto no prettierrc ou algo assim